### ISSUE TITLE
Date is always 01/01/2000

### ISSUE THREAD

**Issue by etatto** [emukidid/swiss-gc#141]
When a game is launched thru SWISS, the date is always 01/01/2000 (and 00:00 for the hour IMHO when starting a game as I have a save file dated as 01/01/2000 1:32). This bug/side-effect could be seen with Mario Kart Double Dash : The save file is always dated as 01/01/2000.
Same happens with:
- Kirby Air Ride
- Mario Party 4 to 7
- Mario Power Tennis
- Sonic Riders
- Super Smash Bros. Melee

**Platform :** GC + Viper GC Extrem + Swiss 4.4 r343

**Comment by suloku** [comment]
It only happens with swiss? Maybe your GC's battery died out.

**Comment by etatto** [comment]
Nope, under Calendar, in the GC settings, date and time are correct, and If I boot my original Mario Kart disc, the save date & time are correct.
After searching the gc-forever forum, I found this : http://www.gc-forever.com/forums/viewtopic.php?t=481, dated back in 2011, I don't know if it was patched...

**Comment by etatto** [comment]
I made a test to confirm it is a SWISS related date/time issue:
- Original disc direct boot => current date and correct time for the save,
- Original disc boot with "Original BIOS" menu under Cobra => current date and correct time for the save,
- Original disc launched thru SWISS => 01/01/2000 date and time starting at 00:00...
- ISO on SD Gecko launched thru SWIS => Same as "Original disc launched thru SWISS"

I don't have any game date-related easter egg (as stated in the gc-forever thread) to test, because I don't know which ones have this, so if someone could give a list of such games, I might be do a test or two ;-)

**Comment by emukidid** [comment]
It's a specific problem with ViperGC since it's leaving the RTC unusable because I must be ignoring certain commands I'm meant to use for it. I've tried without a ViperGC and it's fine - I'll need to dig out my ViperGC cube and have a play.

**Comment by CheatAbuser** [comment]
My ViperGC gamecube has the same issue. Could you solve it? Thanks.

**Comment by antmerino** [comment]
Sorry , don't have any solution? Any advice? Thanks a lot for your work
